scripts/docs/README.md

```diff
@@ -0,0 +1,21 @@
+# Bazel Release Documentation
+
+This directory contains scripts to build the versioned documentation for a new Bazel release.
+
+## Build release documentation
+
+You can build the release documentation by running this command from within a release branch:
+
+```
+bazel build //scripts/docs:gen_release_docs --config=docs
+```
+
+This is only necessary for testing, though. There is a separate pipeline that handles this task for actual Bazel releases.
```

Comment:
Does this mean the separate pipeline uses alternative calls? Or it uses this call from another invoker? Is there any reason for humans to call this (aside from testing the pipeline itself)?

Reply:
It's only for testing.

For release docs the release manager has to invoke another tool, which includes building this target on Kokoro.